ci/dev/vscode.patch

```diff
@@ -248,19 +248,17 @@ index 1e16cde724..0000000000
 -target "12.4.0"
 -runtime "node"
 diff --git a/src/vs/base/common/network.ts b/src/vs/base/common/network.ts
-index e4546b2cf6..9df12239fb 100644
+index e4546b2cf6..ad2c544e89 100644
 --- a/src/vs/base/common/network.ts
 +++ b/src/vs/base/common/network.ts
-@@ -94,16 +94,18 @@ class RemoteAuthoritiesImpl {
+@@ -94,16 +94,17 @@ class RemoteAuthoritiesImpl {
  		if (host && host.indexOf(':') !== -1) {
  			host = `[${host}]`;
  		}
 -		const port = this._ports[authority];
 +		// const port = this._ports[authority];
  		const connectionToken = this._connectionTokens[authority];
--		let query = `path=${encodeURIComponent(uri.path)}`;
-+		// NOTE@coder: Use fsPath for Windows support.
```

Comment:
> We use

Reply:
This is a removed line actually :smile: This is one of the reasons a fork might be better, reviewing changes to the patch is not fun.

Reply:
Oh yea lol.